### PR TITLE
Update Helper to Check for Bundle with correct Extension

### PR DESCRIPTION
**Pull request by pulkitkkr**
## Summary
This PR fixes the bug as described in the Issue: https://github.com/shakacode/react_on_rails/issues/1510

- - -
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/1511)

### CODE REVIEW

File: lib/react_on_rails/helper.rb [hunk 1]
```diff
@@ -109,8 +108,10 @@ def load_pack_for_component(component_name)
       append_stylesheet_pack_tag "generated/#{component_name}"
     end
 
-    def generated_components_pack(component_name)
-      "#{ReactOnRails::WebpackerUtils.webpacker_source_entry_path}/generated/#{component_name}"
+    def generated_components_pack_path(component_name)
+      extension = PacksGenerator.generated_pack_extension
```
Comment (justin808) [inline lib/react_on_rails/helper.rb]: Why is this customizable? Why not always `.js`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ Changes since last non-beta release.
 
 *Please add entries here for your pull requests that are not yet released.*
 - Added `./bin/dev` and `./bin/dev-static` executables to ease and standardize running the dev server. [PR 1491](https://github.com/shakacode/react_on_rails/pull/1491) by [ahangarha](https://github.com/ahangarha)
+- Fixed pack not found warning while using `react_component` and `react_component_hash` helpers, even when corresponding chunks are present in `manifest` file [PR 1511](https://github.com/shakacode/react_on_rails/pull/1511) by [pulkitkkr](https://github.com/pulkitkkr)
```
Comment (justin808) [inline CHANGELOG.md]: what does the manifest have to do with the changes?

File: lib/react_on_rails/helper.rb (hunk 1 above)
Comment (pulkitkkr) [reply lib/react_on_rails/helper.rb]: It's not configurable! It's always js

File: CHANGELOG.md (hunk 2 above)
Comment (pulkitkkr) [reply CHANGELOG.md]: Updated it to: 
```
Fixed pack not found warning while using `react_component` and `react_component_hash` helpers, even when corresponding chunks are present.
```

File: lib/react_on_rails/helper.rb (hunk 1 above)
Comment (justin808) [reply lib/react_on_rails/helper.rb]: Then why not a constant?

**Review by justin808 — CHANGES_REQUESTED** [review]
Why no tests?

**Comment by pulkitkkr** [comment]
Added Tests

**Review by pulkitkkr — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/1511)* status: 0 of 4 files reviewed, 2 unresolved discussions (waiting on @Judahmeek and @justin808)

___
*[`lib/react_on_rails/helper.rb` line 112 at r1](https://reviewable.io/reviews/shakacode/react_on_rails/1511#-NMhBgwlnWkAJz4SH479-r1-112:-NMm8-VvAN3CfOzvXGxb:b-896fix) ([raw file](https://github.com/shakacode/react_on_rails/blob/5fb4f352bc114bb419cfebf5216af0d0ea83e39e/lib/react_on_rails/helper.rb#L112)):*
<details><summary><i>Previously, justin808 (Justin Gordon) wrote…</i></summary><blockquote>

Then why not a constant?
</blockquote></details>

Done.

**Review by justin808 — APPROVED** [review]

File: lib/react_on_rails/helper.rb [hunk 3]
```diff
@@ -109,8 +108,10 @@ def load_pack_for_component(component_name)
       append_stylesheet_pack_tag "generated/#{component_name}"
     end
 
-    def generated_components_pack(component_name)
-      "#{ReactOnRails::WebpackerUtils.webpacker_source_entry_path}/generated/#{component_name}"
+    def generated_components_pack_path(component_name)
```
Comment (justin808) [inline lib/react_on_rails/helper.rb:111]: Should this be part of the helper? or not?

File: lib/react_on_rails/helper.rb [hunk 4]
```diff
@@ -94,8 +94,7 @@ def react_component(component_name, options = {})
     end
 
     def load_pack_for_component(component_name)
```
Comment (justin808) [inline lib/react_on_rails/helper.rb:96]: why is this part of the public helper?

there's no doc for this method `load_pack_for_component` 
or `generated_components_pack_path`

Comment (pulkitkkr) [reply lib/react_on_rails/helper.rb:96]: It should be private member as it is used internally by `react_component` and `react_component_hash`

File: lib/react_on_rails/helper.rb (hunk 3 above)
Comment (pulkitkkr) [reply lib/react_on_rails/helper.rb:111]: It should be a private member

**Pull request merged**